hvantk/utils/ucsc.py

```
@@ -0,0 +1,152 @@
+
+    # Convert Pandas DataFrame to Hail Table with the specified key
+    ht = hl.Table.from_pandas(df, key=index_name)
+
+    return ht
+
+
+def create_mt_from_ucsc_expression_matrix(
+        expression_matrix_path: str,
+        output_path: str = None,
+        delimiter: str = "\t",
+        row_fields: dict = {UCSC_GENE_COLUMN: hl.tstr},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)